.github/workflows/posix.yml

```diff
@@ -0,0 +1,14 @@
+name: POSIX Builds
+workflow_dispatch:
+    inputs:
+        foo:
+            description: 'placeholder'
+            default: 'bar'
+            required: 'false'
+
+jobs:
+    hello_world:
+    runs-on: ubuntu-latest
```

Comment:
This is a problematic line 🤔  :) https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/actions/runs/1751437730

Reply:
```
Invalid workflow file : .github/workflows/posix.yml#L11
You have an error in your yaml syntax on line 11
```

Reply:
This is all going to get torn out anyway.